Process.c

```diff
@@ -742,8 +742,8 @@ void Process_printPercentage(float val, char* buffer, int n, uint8_t width, int*
       } else {
          *attr = CRT_colors[PROCESS_MEGABYTES];
          if (val < 100.0F)
-            val = 100.0F; // Don't round down and display "val" as "99".
-         xSnprintf(buffer, n, "%*.0f ", width, val);
+            val = 100.0F; // Don't round down and display "val" lower than "100.0".
+         xSnprintf(buffer, n, "%*.1f ", width, val);
       }
```

Comment:
If both branches of this if end up with the same number format string, we probably come better with restructuring the flow of this `if` statement to have the first part handle coloring and the second part the formatting.